<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MiscellaneousWidget</class>
 <widget class="QWidget" name="MiscellaneousWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>613</width>
    <height>426</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout">
   <item row="0" column="0">
    <layout class="QGridLayout">
     <item row="2" column="0">
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>Countries</string>
       </property>
       <layout class="QGridLayout">
        <item row="0" column="0">
         <layout class="QVBoxLayout">
          <item>
           <widget class="QTableView" name="countriesTable"/>
          </item>
          <item>
           <layout class="QHBoxLayout">
            <item>
             <widget class="QPushButton" name="pushButton_new">
              <property name="text">
               <string/>
              </property>
              <property name="icon">
               <iconset>
                <normaloff>:/images/images/adept_install.png</normaloff>:/images/images/adept_install.png</iconset>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton_delete">
              <property name="text">
               <string/>
              </property>
              <property name="icon">
               <iconset>
                <normaloff>:/images/images/adept_purge.png</normaloff>:/images/images/adept_purge.png</iconset>
              </property>
             </widget>
            </item>
            <item>
             <spacer>
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton_revert">
              <property name="text">
               <string/>
              </property>
              <property name="icon">
               <iconset>
                <normaloff>:/images/images/button_cancel.png</normaloff>:/images/images/button_cancel.png</iconset>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton_submit">
              <property name="text">
               <string/>
              </property>
              <property name="icon">
               <iconset>
                <normaloff>:/images/images/button_ok.png</normaloff>:/images/images/button_ok.png</iconset>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item row="4" column="0">
      <spacer>
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="4" column="1">
      <spacer>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="2" column="1">
      <widget class="QGroupBox" name="logBooks">
       <property name="title">
        <string>Log Books</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0">
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="0" column="0">
           <widget class="QListView" name="logBookTable"/>
          </item>
          <item row="1" column="0">
           <layout class="QGridLayout" name="gridLayout">
            <item row="0" column="0" colspan="2">
             <widget class="QLabel" name="book_label">
              <property name="text">
               <string>Boo&amp;k</string>
              </property>
              <property name="buddy">
               <cstring>book_spinBox</cstring>
              </property>
             </widget>
            </item>
            <item row="0" column="2">
             <widget class="QSpinBox" name="book_spinBox">
              <property name="buttonSymbols">
               <enum>QAbstractSpinBox::PlusMinus</enum>
              </property>
             </widget>
            </item>
            <item row="0" column="3">
             <widget class="QLabel" name="lines_label">
              <property name="text">
               <string>Lines p.P.</string>
              </property>
              <property name="buddy">
               <cstring>lines_spinBox</cstring>
              </property>
             </widget>
            </item>
            <item row="0" column="4">
             <widget class="QSpinBox" name="lines_spinBox">
              <property name="readOnly">
               <bool>false</bool>
              </property>
              <property name="buttonSymbols">
               <enum>QAbstractSpinBox::PlusMinus</enum>
              </property>
             </widget>
            </item>
            <item row="1" column="0" colspan="2">
             <widget class="QLabel" name="start_label">
              <property name="text">
               <string>Start Page</string>
              </property>
              <property name="buddy">
               <cstring>start_spinBox</cstring>
              </property>
             </widget>
            </item>
            <item row="1" column="2">
             <widget class="QSpinBox" name="start_spinBox">
              <property name="buttonSymbols">
               <enum>QAbstractSpinBox::PlusMinus</enum>
              </property>
              <property name="maximum">
               <number>999</number>
              </property>
             </widget>
            </item>
            <item row="1" column="3">
             <widget class="QLabel" name="end_label">
              <property name="text">
               <string>End Page</string>
              </property>
              <property name="buddy">
               <cstring>end_spinBox</cstring>
              </property>
             </widget>
            </item>
            <item row="1" column="4">
             <widget class="QSpinBox" name="end_spinBox">
              <property name="buttonSymbols">
               <enum>QAbstractSpinBox::PlusMinus</enum>
              </property>
              <property name="maximum">
               <number>999</number>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="layout_label">
              <property name="text">
               <string>La&amp;yout</string>
              </property>
              <property name="buddy">
               <cstring>comboBox</cstring>
              </property>
             </widget>
            </item>
            <item row="2" column="1" colspan="4">
             <widget class="QComboBox" name="comboBox"/>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QPushButton" name="pushButton">
       <property name="text">
        <string>PushButton</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
